docker/test-amazonlinux2.dockerfile

```diff
@@ -0,0 +1,25 @@
+ARG base_image=swift:5.8-amazonlinux2
+FROM $base_image
+# needed to do again after FROM due to docker limitation
+ARG swift_version
+ARG ubuntu_version
+
+# dependencies
+RUN yum install -y \
+    curl \
+    gcc \
+    gcc-c++ \
+    make
+COPY ./scripts/install-libarchive.sh /
+RUN /install-libarchive.sh
+
+# tools
+RUN mkdir -p $HOME/.tools
+RUN echo 'export PATH="$HOME/.tools:$PATH"' >> $HOME/.profile
+
+# swiftformat (until part of the toolchain)
+
+ARG swiftformat_version=0.49.18
```

Comment:
Why is SwiftFormat installed in the amazonlinux and ubi9 Dockerfiles but not the standard test Dockerfile

Reply:
Oh good catch, that was just an oversight. We have the separate lint/soundness dockerfile, so the test dockerfiles don't need to install swiftformat anymore. Removed.